cmd/protoc-gen-go-tetragon/common/common.go

```diff
@@ -16,6 +16,9 @@ import (
 // TetragonPackageName is the import path for the Tetragon package
 var TetragonPackageName = "github.com/cilium/tetragon"
 
+// EventcheckerPackageName is the import path for the Eventchecker package
+var EventcheckerPackageName = "github.com/cilium/"
```

Comment:
This isn't the full path to the eventchecker package. Maybe the variable could have a different name?

Reply:
I think its unused so can just be removed. It was a hold out from early version.